docs/patterns/navigation.md

```diff
@@ -45,6 +45,12 @@ or more `p-subnav` components.
 View example of the sub-navigation pattern
 </a>
 
+<div class="p-notification--caution">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Warning:</span>This pattern requires JS to be functional.
+  </p>
+</div>
```

Comment:
Again we should explain the attributes which need to be toggled on click

Reply:
Added 👍